Code/GeorgiaLibrarySystem-/GtlService/GtlService.csproj

```diff
@@ -146,11 +146,23 @@
       </FlavorProperties>
     </VisualStudio>
   </ProjectExtensions>
+  <PropertyGroup>
+    <PreBuildEvent>
+    </PreBuildEvent>
+  </PropertyGroup>
+  <PropertyGroup>
+    <PostBuildEvent>
+    </PostBuildEvent>
+  </PropertyGroup>
   <!-- To modify your build process, add your task inside one of the targets below and uncomment it. 
        Other similar extension points exist, see Microsoft.Common.targets.
   <Target Name="BeforeBuild">
   </Target>
   <Target Name="AfterBuild">
   </Target>
   -->
+  <Target Name="OnCleanOnly" AfterTargets="Clean">
+    <Exec Command="&#xD;&#xA;	echo ufo&#xD;&#xA;                   cd ..\..\Database&#xD;&#xA;	call ScriptDatabase.bat&#xD;&#xA;                   " />
+    <Message Text="UFO invasion detected" Importance="High" />
```

Comment:
what is this...

Reply:
pre-build action. it makes it run the sql script before going on forward with normal visual studio/c# stuff. also prints UFO invasion detected in the output console :D